examples/next/pages/ui/components/liveness/components/LivenessDefault.tsx

```diff
@@ -1,25 +1,35 @@
+import React from 'react';
 import { View, Flex, Loader, Text } from '@aws-amplify/ui-react';
 import { FaceLivenessDetectorCore } from '@aws-amplify/ui-react-liveness';
 import { useLiveness } from './useLiveness';
+import { ChallengeSelection } from './ChallengeSelection';
 import { SessionIdAlert } from './SessionIdAlert';
 import LivenessInlineResults from './LivenessInlineResults';
 
+const DEFAULT_CHALLENGE = 'FaceMovementAndLightChallenge';
+const SUPPORTED_CHALLENGES = [
+  'FaceMovementAndLightChallenge',
+  'FaceMovementChallenge',
+];
+
 export default function LivenessDefault({
-  disableStartScreen = false,
   components = undefined,
   credentialProvider = undefined,
+  disableStartScreen = false,
 }) {
+  const [challengeType, setChallengeType] = React.useState(DEFAULT_CHALLENGE);
+
   const {
     getLivenessResponse,
     createLivenessSessionApiError,
     createLivenessSessionApiData,
     createLivenessSessionApiLoading,
     handleGetLivenessDetection,
     stopLiveness,
-  } = useLiveness();
+  } = useLiveness(challengeType);
```

Comment:
Not a blocker: how does this work with connection opening on the camera check screen? Would we need to re-open a request each time we change the challenge type?

Reply:
Yeah it creates the session with the default `FaceMovementAndLight` but then if you change the selection it calls `mutate` with the start key and creates a new session with the new challenge. I believe this basically marks the data as invalid so makes the same api call again. I'm not super familiar with `swr` so definitely open to cleaner/better ways of doing that though. 

https://github.com/aws-amplify/amplify-ui/assets/70536670/9d347a0d-5e69-46cd-afc6-1b4c0d265ce7